Cargo.toml

```diff
@@ -59,7 +59,7 @@ strum_macros = ">= 0.16, <= 0.18"
 snafu = ">= 0.5, <= 0.6"
 getset = "0.0.9"
 enum-map = "0.6"
-triple_accel = "0.3.4"
+triple_accel = "0.3.2"
```

Comment:
Why the revert? Did 0.3.4 break something?
(since I'd love to just omit the patch number, since that's for... patches :D)

Reply:
Since it's off-topic here I made it into another PR: https://github.com/rust-bio/rust-bio/pull/364